docs/source/config/settings.rst

```diff
@@ -331,6 +331,14 @@ Customization
 
     Default: ``None``
 
+.. data:: ABSOLUTE_PASS_LOGO_PATH
```

Comment:
I'd prefer to go the same way like we did for the wallet (`WALLET_LOGO_URL` in the config, `ABSOLUTE_...` as a property)

Reply:
Using `WALLET_LOGO_URL` as from Matrix discussion.